tests/integrationv2/providers.py

```diff
@@ -113,6 +113,9 @@ class S2N(Provider):
     """
     def __init__(self, options: ProviderOptions):
         self.ready_to_send_input_marker = None
+        self.tls12_pq_cipher_prefs = [Ciphers.KMS_PQ_TLS_1_0_2019_06, Ciphers.PQ_SIKE_TEST_TLS_1_0_2019_11,
+                                      Ciphers.KMS_PQ_TLS_1_0_2020_07, Ciphers.KMS_PQ_TLS_1_0_2020_02,
+                                      Ciphers.PQ_SIKE_TEST_TLS_1_0_2020_02]
```

Comment:
Why make the list an instance variable?

I'd suggest a global (named in all caps to indicate it's a constant), either in this file or imported from common.py.

Reply:
Updated